website/src/components/Chat/ChatConversation.tsx

```diff
@@ -119,27 +111,28 @@ export const ChatConversation = memo(function ChatConversation({ chatId: chatIdP
         .find((m) => m.role === "assistant" && m.state === "complete")?.id ?? null;
 
     const prompter_arg: InferencePostPrompterMessageParams = {
-      chat_id: currentChatId,
+      chat_id: chatId,
       content,
       parent_id,
     };
 
     const prompter_message: InferenceMessage = await post(API_ROUTES.CREATE_PROMPTER_MESSAGE, { arg: prompter_arg });
-    mutate(API_ROUTES.LIST_CHAT); // revalidte chat list after create prompter message to make sure the message already has title
```

Comment:
To prevent refresh, we simply can check whether the chatIdProps is null or not here.

Reply:
`chatIdProps` will never be null in this new version, which is why I check the messages.

Reply:
I mean you can do this in the old version. I will switch to the current UI after this anyway since that much better UI. Btw I would like to have #2484 merge first because this PR may cause a lot of conflict with it.